src/content/guides/tree-shaking.md

```diff
@@ -107,23 +108,46 @@ function cube(x) {
 Note the `unused harmony export square` comment above. If you look at the code below it, you'll notice that `square` is not being imported, however, it is still included in the bundle. We'll fix that in the next section.
 
 
-## Minify the Output
+## Mark the file as side-effect-free
+
+In a 100% ESM module world, identifying side effects is straightforward. However, we don't live in that reality just yet, so it's necessary to provide hints to webpack's compiler on the "pureness" of your code.
```

Comment:
Can you rephrase `we don't live in that reality just yet, so` to `we aren't there just yet, in the mean time`